### DISCUSSION TITLE
임주연 1회차 블로그 피드백 요청 드립니다 🙏🏻

### DISCUSSION THREAD

**Discussion by juuyeoon** [Orm-camp/blog-feedback#2] [동글동글 2기 피드백]
[https://juuyeoon.github.io/gen_ai/proj_gen_ai_1/](url)

이전에 진행했던 프로젝트를 기록해 두고자 작성한 글입니다.
부족한 부분이 많은 글이라, 적극적인 피드백 대환영입니다 🙇🏻‍♀️

**Comment by Orm-camp** [comment]
글 잘 읽었습니다!
제안 드리고 싶은 부분이 두 가지 있습니다.

1. Whisper AI API를 어떻게 쓰시는지 활용이 궁금해집니다. 다음 글에서 확인할 수 있을까요?
2. 소제목으로 구분된 문단끼리의 거리가 너무 가까운 것 같습니다. 조금 더 구분될 수 있도록 공간을 주는 것도 필요하다고 생각됩니다.

글 쓰시느라 고생하셨어요 😊

**Reply by juuyeoon** [reply]
피드백 감사합니다!

1. Whisper AI API를 어떻게 가지고 와서 쓰는지 그 방법에 대한 내용은 작성할 계획이 없었는데, 독자의 니즈(!)가 있다면 작성해 봐야겠네요.
2. 문단끼리의 거리가 조금 더 벌어져서 확연하게 구분될 수 있도록 간격을 더 주거나, 다른 팁이 있는지 찾아보겠습니다! 혹시 좋은 팁 있으시다면 디스코드에서 공유해 주세요 :)
